Fix sourcing filenames (#691)
MicrosoftDocs/vscodespaces: vscodespaces/reference/personalizing.md

If none of these files are found, any files and folders starting with `.` are symlinked to the home (`~` or `$HOME` on Linux, `%HOMEPATH%` on Windows) directory. In addition, a couple of files are picked up automatically by convention for sourcing.

On Linux-based codespaces:
  - `zshrc` for zsh shell
  - `.bashrc` and `.bashprofile` for Bash and other shell

On Windows-based codespaces:

>   - `profile.ps1` for PowerShell, this is not actually sourced as on Linux, but rather copied to `%USERPROFILE%\Documents\WindowsPowerShell\profile.ps1`

Reviewer: ```suggestion
  - `profile.ps1` for PowerShell
    - **Note:** This file is copied to `%USERPROFILE%\Documents\WindowsPowerShell\profile.ps1` instead of being sourced.
```
